presto-main/src/main/java/com/facebook/presto/cost/PlanNodeStatsEstimateMath.java

```diff
@@ -148,7 +148,7 @@ public static PlanNodeStatsEstimate intersect(PlanNodeStatsEstimate left, PlanNo
                             .setNullsFraction(min(leftSymbolStats.getNullsFraction(), rightSymbolStats.getNullsFraction()))
                             .build());
         }
-
+        statsBuilder.setOutputRowCount(left.getOutputRowCount() + right.getOutputRowCount());  // this is the maximum row count;
```

Comment:
it's needed because otherwise the input to the group by doesn't have a row count (and then the output size is estimated at NaN)